src/components/calcite-list-item/calcite-list-item.tsx

```diff
@@ -0,0 +1,152 @@
+import { Component, Element, Event, EventEmitter, Prop, h, VNode } from "@stencil/core";
+import { SLOTS, CSS } from "./resources";
+import { getSlotted } from "../../utils/dom";
+
+/**
+ * @slot actions-start - todo document.
```

Comment:
JSDoc should be filled out before merging since we have automated doc-update PRs enabled.